gmojiAndroid/build.gradle

```diff
@@ -102,7 +125,6 @@ model {
 
                     cppFlags.addAll(coreCppFlags(abi, ofRoot()))
                     cppFlags.addAll(addonCppFlags(abi, ofRoot()))
-                    cppFlags.add("-O2")

```

Comment:
I think this line was very important for performance. DLIB performance depends very much on optimization